lib/selinux/selinux_linux.go

```diff
@@ -86,9 +90,39 @@ func FileContexts(dataDir, configPath string) (string, error) {
 	return buf.String(), nil
 }
 
+type selinuxContext struct {
+	user   string
+	role   string
+	domain string
+	mls    string
```

Comment:
What's mls? Should we just call it level?

Reply:
Stands for multi level security, is more accurate than what the dep uses as "level"